Bixal/move.mil
# Feature/discounts improvements

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

Refactored the parsing of the discounts file. It looks for all the discounts .csv.enc files in the 'lib/data' dir. Reads and decrypts all of them and pushes the data to the database.

## Testing

1. Pull Changes.
2. Install the module.
3. Run 'docker-compose run php drupal parser'
4. Check the inserted data.
        docker-compose run php drupal database:client
        select * from parser_discounts;
## Screenshots

_Attach relevant before and after screenshots here._
